data/tribes/initialization/frisians/units.lua

```diff
@@ -1971,7 +1977,7 @@ wl.Descriptions():new_tribe {
             -- TRANSLATORS: Purpose helptext for a Frisian production site: Reindeer Farm
             purpose = pgettext("frisians_building", "Breeds strong reindeer for adding them to the transportation system. Also keeps them for their fur, which is turned into armor and cloth."),
             -- TRANSLATORS: Note helptext for a Frisian production site: Reindeer Farm
-            note = pgettext("frisians_building", "If the supply is steady, the reindeer farm produces one meat after producing three pieces of fur.")
+            note = pgettext("frisians_building", "If this building is fully supplied and both reindeers and furs are needed by the economy, production of one reindeer takes %1$s, one fur takes %2$s and one meat takes %3$s on average. If only reindeers are needed by the economy, their production takes %4$s on average. If only fur is needed by the economy, production of one fur takes %5$s and one meat takes %6$s on average."):bformat(format_minutes_seconds(1, 10), format_minutes_seconds(1, 10), format_minutes_seconds(3, 29), format_seconds(30), format_seconds(40), format_minutes_seconds(1, 59))
```

Comment:
I think this needs different treatment. Average only fur and reindeer for the full cycle text, and say that "in addition, one meat is produced each <cycle_time>". And similarly for the fur only case.

Reply:
What is reason for different treatment? With the "in addition..." it makes text even larger.
The original proposed performance text should be technically correct, and gives user comparable information e.g. towards Hunter's House production of meat.
There is one hidden information, probably you know - the meat itself can't be set to be needed by economy.

Reply:
Yes, the target setting is the big difference, especially since it's only done for 3rd fur, so it may be skipped even when some fur is needed.

The meat is only a by-product point is always mentioned for the reindeer farm, both in the Frisian campaign and in the [forum](https://www.widelands.org/forum/post/37363/).

Reply:
sorry to request another change, I just noticed you changed the "note" helptext instead of introducing a performance helptext. 
So as we have to move the performance part to a "performance" section anyway my proposal would be to mention only reindeer and fur in Performance part and have the "note" helptext explaining that meat is a byproduct of each 3rd steadily produced fur.  
@tothxa

Reply:
I kept the original note text, which seems good enough. Performance text now without meat.